### ISSUE TITLE
Bug: Core Data code generation isn't supported

### ISSUE THREAD

**Issue by brentleyjones** [buildbuddy-io/rules_xcodeproj#395]
## Describe the bug

https://github.com/buildbuddy-io/rules_xcodeproj/pull/288#issuecomment-1098303363

**Comment by brentleyjones** [comment]
We now [process Info.plist in Bazel](https://github.com/buildbuddy-io/rules_xcodeproj/blob/c20d4760635faddd2437b7991d5ec69b39aa1d83/xcodeproj/internal/info_plists.bzl#L16-L38), which can serve as an example of how to do this.

**Comment by brentleyjones** [comment]
https://github.com/buildbuddy-io/rules_xcodeproj/commit/cf20e53117780eb0912a3314948bccc83af872bf is another example of how we do some post-processing in Bazel for use in Xcode.

**Comment by brentleyjones** [comment]
I haven't heard additional complaints about this, so bumping it out of blocking 1.0.

**Comment by wileykestner** [comment]
👋 Hey @brentleyjones I think we just hit this one… we are preparing to generate:

```
OurSpecialModel.xcdatamodeld
└── 1.0.xcdatamodel
    └── contents // <-- we are generating this file
```

The specific error I saw when trying to generate the project (BwB):

```
ERROR: Internal precondition failure:
"path/to/OurSpecialModel.xcdatamodeld" `XCVersionGroup` not found in `elements`
Please file a bug report at https://github.com/MobileNativeFoundation/rules_xcodeproj/issues/new?template=bug.md
```

Please feel free to re-direct if this error doesn't seem related to this issue.  And thanks for such an amazing tool!

**Comment by brentleyjones** [comment]
This is a different error. Codegen not being supported is referred to BwX mode Xcode code generation of core data classes.

Can you file a new issue, with a repro, and I'll look into a fix. I think I know what the root cause of it is.

**Comment by wileykestner** [comment]
Absolutely, thanks for the redirect.  I'll see if I can work up a repro.

**Comment by wileykestner** [comment]
Oof - now that we resolved our BwB core model generation issue of course we ran headlong into this one.

+1 on this issue in case anyone is tracking relative priority.

**Comment by brentleyjones** [comment]
With BwX mode going away, this issue might become moot.
